registrations/forms.py

```diff
@@ -0,0 +1,31 @@
+from django import forms
+
+from registrations.models import RegistrationPriceGroup
+
+
+class RegistrationPriceGroupAdminForm(forms.ModelForm):
+    price = forms.DecimalField(
+        required=False, max_digits=19, decimal_places=2, min_value=0
+    )
+    price_without_vat = forms.DecimalField(
+        required=False, disabled=True, max_digits=19, decimal_places=2, min_value=0
+    )
+    vat = forms.DecimalField(
+        required=False, disabled=True, max_digits=19, decimal_places=2, min_value=0
+    )
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        if not self.instance.pk:
+            self.initial["vat_percentage"] = RegistrationPriceGroup.VatPercentage.VAT_24
+
+    class Meta:
+        model = RegistrationPriceGroup
+        fields = (
+            "price_group",
+            "price",
+            "vat_percentage",
+            "price_without_vat",
+            "vat",
+        )
```

Comment:
Would it be possible to return PriceGroup `description` for the RegistrationPriceGroup as well?

At the moment price group description is only available via signup_group and requires extra requests to show in the UI. It makes the UI implementation a little bit more complicated. Also price_group endpoint requires authentication so price group description is not possible to show for unauthenticated users